geometry_msgs/PoseWithCovarianceStamped poseIn
---
geometry_msgs/PoseWithCovarianceStamped poseOut